Tests/Controllers/PeopleController.cs

```diff
@@ -44,6 +45,77 @@ public IEnumerable<Person> PaginateAndQueryPeople()
             return GetPeopleNotRandom();
         }
 
+        [HttpGet]
+        [AllowsManuallyHandledQuery]
+        [Paginated]
+        [DisableDefaultIncluded]
```

Comment:
Does `DisableDefaultIncluded` make sense in combination with `ManuallyHandlesQuery`? At first glance it does not and maybe it should error (?). If that's hard then IMO `ManuallyHandlesQuery` should take precedence and all other query-related attributes should be ignored.

Reply:
That makes sense. I've added validation to `HandlesQueryAttribute`, so it will return an error when any\both `DisableDefaultIncluded` \ `AllowsQuery` are specified.